lib/src/df.dart

```diff
@@ -51,33 +52,34 @@ class DataFrame {
       List<String>.from(_columns.map<String>((c) => c.name));
 
   /// The dataframe columns indices
-  Map<int, String> get columnsIndices => _columnsIndices();
```

Comment:
I don't think this data structure needed to be a map-I've changed it to a list to improve readability.